.github/workflows/ci.yml

```diff
@@ -242,8 +244,6 @@ jobs:
         python -m pip install -r tests/requirements.txt
 
     - name: Configure
-      env:
-        SETUPTOOLS_USE_DISTUTILS: stdlib
```

Comment:
Testing this - this is invalid in 3.12, there is no stdlib distutils, and we shouldn't be using before that either. Just looking to see what fails.

Reply:
FYI, I expect this was to build numpy before wheels were available.